samples/maskededit/maskedctrl.cpp

```diff
@@ -0,0 +1,713 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        samples/maskededit/maskedctrl.cpp
+// Purpose:     wxWidgets masked edit control sample
+// Author:      Manuel Martin
+// Modified by:
+// Created:     2012-07-30
+// RCS-ID:      $Id$
+// Copyright:   (c)
```

Comment:
```suggestion
// Created:     2012-07-30
// Copyright:   (c) 2012 Manuel Martin
```
or something like this.